src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue

```diff
@@ -0,0 +1,178 @@
+              {{ trans('flow-builder.select-value-for-choices-for-selected-property') }}
+            </div>
+            <div v-else>
+              {{ trans('flow-builder.enter-value-for-choices-for-selected-property') }}
+            </div>
+
+            <div
+              v-for="choiceKey in choiceKeys"
+              :key="choiceKey"
+              class="mt-2">
+              <label>{{ trans('flow-builder.choice') }}: {{ choiceKey }}</label>
+              <vue-multiselect
+                v-if="isMultipleChoiceProperty"
+                :value="getChoiceValueOption(choiceKey)"
+                :options="choiceValueOptions"
+                :show-labels="false"
+                :placeholder="trans('flow-builder.select-a-value')"
+                track-by="value"
+                label="description"
+                @input="setChoiceValueOption($event, choiceKey)" />
+              <input
+                v-if="isTextProperty"
+                :value="getChoiceValue(choiceKey)"
+                type="text"
+                class="form-control"
+                :placeholder="trans('flow-builder.enter-value')"
+                @input="setChoiceValue($event.target.value, choiceKey)">
+              <input
+                v-if="isNumberProperty"
+                :value="getChoiceValue(choiceKey)"
+                type="number"
+                class="form-control"
+                :placeholder="trans('flow-builder.enter-value')"
+                @input="setChoiceValue(Number($event.target.value), choiceKey)">
+            </div>
+          </template>
+        </div>
+      </template>
+    </generic-contact-property-editor>
+  </div>
+</template>
+
+<script lang="ts">
+import VueMultiselect from 'vue-multiselect'
+import Component, {mixins} from 'vue-class-component'
+import {Prop} from 'vue-property-decorator'
+import Lang from '@/lib/filters/lang'
+import {ISelectOneResponseBlock} from '@floip/flow-runner/src/model/block/ISelectOneResponseBlock'
+import {IContactPropertyOption} from '@/store/flow/block-types'
+import {Getter, namespace} from 'vuex-class'
+import {find} from 'lodash'
+import {IContactPropertyMultipleChoice} from '@/store/flow/block-types/Core_SetContactPropertyStore'
+import {choicesToExpression} from './expressionTransformers'
+
+const flowVuexNamespace = namespace('flow')
+
+@Component({
+  components: {
+    VueMultiselect,
+  },
+})
+export class SelectOneResponseBlockContactPropertyEditor extends mixins(Lang) {
+  @Prop() readonly block!: ISelectOneResponseBlock
+
+  get contactPropertyName(): string | undefined {
+    return this.block.config.set_contact_property?.[0].property_key
+  }
+
+  get contactProperty(): IContactPropertyOption | null {
+    return find(this.subscriberPropertyFields, contactProperty => contactProperty.name === this.contactPropertyName) ?? null
+  }
+
+  get isMultipleChoiceProperty(): boolean {
+    return this.contactProperty?.data_type === 'multiple_choice'
```

Comment:
From the AC (based on last discussion we had with George), for some runtime complexity reason, we've agreed
> We will cover only **text** & **number** contact property types. The other ones will be disabled in the dropdown.

So I think, we should not cover data_type === 'multiple_choice'

Reply:
We should have removed all multiple_choice reference form AC/miro doc to avoid this misunderstanding

Reply:
I remember seeing this in the ticket description, but the newer designs had those dropdown menus for each choice, which confused me, I guess.

Reply:
I see, next time, let's make sure we clarify confusing part to avoid extra work ;)

Reply:
Fixed here: https://github.com/FLOIP/flow-builder/pull/302/commits/55c3f519b0f6fb9c4b4a1810771d6e07589bff9a
If we need to enable `multiple_choice` contact properties again in the future, we can basically undo this commit.